### DISCUSSION TITLE
NJPilot, log of new 1.0.4 plugin install

### DISCUSSION THREAD

**Discussion by NJTravelerDH** [calibx/msfsdeck#129] [Q&A] [answered]
I think the problem is I cannot double-click the file while I am in User space.  Windows states "How do you want to open this file? " and nothing else happens.  I can double-click it in ADMIN space.  It runs quickly with no error.   

After the "install" (in DHAdmin):
The user "David" has a MSFS file  => C:\Users\David\AppData\Local\Loupedeck\Plugins has a MSFS folder (dated 9/29/22)
and C:\Users\David\AppData\Local\Loupedeck\Logs\plugin_crashes folder is empty.
BUT
the admin "DHAdmin" has  C:\Users\DH Admin\AppData\Local\Loupedeck\Logs
with the file 2022-10-09T19-39-14-0646_LoupedeckServiceTool.log  ,== attached below
and the folder C:\Users\DH Admin\AppData\Local\Loupedeck\Plugins has a Spotify folder (Dated 10/5/22)
[2022-10-09T19-39-14-0646_LoupedeckServiceTool.log](https://github.com/calibx/msfsdeck/files/9742848/2022-10-09T19-39-14-0646_LoupedeckServiceTool.log)

**Comment by calibx** [comment, marked as answer]
Hi,
I dont't exactly know how does the loupedeck ui works with admin. But a simple solution would be to copy MSFS folder in DH Admin plugins folder.
Hope it will works.

**Comment by NJTravelerDH** [comment]
I have successfully installed the Loupedeck Marketplace 2.1.0 plugin.  I beleive the problem in the previous install was that I was not running the Loupdeck software when I installed.  This time, I DID RUN the software (thanks Simhanger).  Everything worked perfectly.  I am excited to try out your additional options.  Thank you very much

**Comment by calibx** [comment]
Great news !
